src/sentry/incidents/logic.py

```diff
@@ -97,6 +97,9 @@ def create_incident(
                     sender=type(incident_project), instance=incident_project, created=True
                 )
 
+        create_incident_activity(
+            incident, IncidentActivityType.STARTED, user=user, date_added=date_started
+        )
         create_incident_activity(incident, IncidentActivityType.DETECTED, user=user)
```

Comment:
Why does create_incident accept a date_detected and uses it to create the Incident, but doesn't use it as the date_added for the detected activity item?